docs/licensing.md

```diff
@@ -32,13 +32,13 @@ In other words, the following will remain part of lakeFS:
   - Integration to publicly available applications accessing an object storage
   - Clear CLI, API and GUI interfaces
   
-We also commit that lakeFS will be scalable in throughout performance, and will support multi region architecture.
+We also commit that lakeFS will be scalable in throughput and performance, and will support multi region architecture.
```

Comment:
"lakeFS will be scalable in throughput, performance, and ?"
do we like two 'and's?

Reply:
I think the two "and"s are ok here. They are referring to two different things.

Reply:
Hurray for the Oxford comma!